[ZH] Update .osr (file format) (#3759)
ppy/osu-wiki: wiki/osu!_File_Formats/Osr_(file_format)/zh.md

> ---
> outdated: false
> ---
> 

Reviewer: ```suggestion
```

> ---
> outdated: false
> ---
> 

Reviewer: ```suggestion
```
标签可以直接删除

| Short | 结算界面显示的最大连击数 |
| Byte | Perfect/full combo (1 = 没有Miss和断滑条) |
| Integer | 使用的Mods。参阅下方的Mods值列表。 |
| String | 生命值图像：用逗号分隔的u/v组，u代表此时的时间（毫秒），v是一个Float浮点值（从0到1）表示此时的生命值（0=空生命值条，1=满生命值条）。 |
| Long | 时间戳 ([Windows ticks](http://msdn.microsoft.com/en-us/library/system.datetime.ticks%28v=vs.110%29.aspx)) |
| Integer | 压缩后的回放数据的长度（用 byte 表示）（原文：Length in bytes of compressed replay data） |
| Byte 数组 | 压缩后的回放数据 |

> 剩余的数据包括在一个 wikipedia:LZMA 流里的鼠标移动与按键的信息。

Reviewer 2: ```suggestion
剩余的数据包括在一个 [LZMA](https://zh.wikipedia.org/wiki/LZMA) 流里的鼠标移动与按键的信息。
```

| Integer | 使用的Mods。参阅下方的Mods值列表。 |
| String | 生命值图像：用逗号分隔的u/v组，u代表此时的时间（毫秒），v是一个Float浮点值从（从0到1）表示此时的生命值（0=空生命值条，1=满生命值条）。 |
| Long | 时间戳 ([Windows ticks](http://msdn.microsoft.com/en-us/library/system.datetime.ticks%28v=vs.110%29.aspx)) |

> | Integer | 压缩后的回放数据的长度（用byte表示）（原文：Length in bytes of compressed replay data） |

Reviewer: ```suggestion
| Integer | 压缩后的回放数据的长度（用 byte 表示）（原文：Length in bytes of compressed replay data） |
```

如果可以，中英文之间应添加空格

| w | Long | 距离上个动作的时间（毫秒） |
| x | Float | 鼠标的X坐标（从0到512） |
| y | Float | 鼠标的Y坐标（从0到384） |
| z | Integer | 鼠标、键盘按键的组合 (M1 = 1, M2 = 2, K1 = 4, K2 = 8, 烟雾 = 16) (K1总是与M1一起使用，K2总是与M2一起使用。1+4=5 2+8=10。) |

> 在`20130319`版本或之后版本创建的回放内，用于分数的32位随机整数seed会以 `-12345|0|0|seed` 的格式编码进LZMA流末尾的附加回放帧中。（原文：the 32-bit integer RNG seed used for the score will be encoded into an additional replay frame at the end of the LZMA stream, under the format `-12345|0|0|seed` .）

Reviewer: ```suggestion
在`20130319`版本或之后版本创建的回放内，用于分数的32位随机整数seed会以 `-12345|0|0|seed` 的格式编码进LZMA流末尾的附加回放帧中。（原文：the 32-bit integer RNG seed used for the score will be encoded into an additional replay frame at the end of the LZMA stream, under the format `-12345|0|0|seed`.）
```
多余的空格。如果可以，`` `20130319` `` 与中文间也应添加空格

Author: 感谢斧正，受教了

| DoubleTime | 64 (6) |  |
| Relax | 128 (7) |  |
| HalfTime | 256 (8) |  |

> | Nightcore | 512 (9) | 总是和DT一起使用 : 512 + 64 = 576 |

Reviewer: ```suggestion
| Nightcore | 512 (9) | 总是和 DT 一起使用 : 512 + 64 = 576 |
```
如果可以，中英文之间应添加空格
